src/components/Facility/PatientNoteCard.tsx

```diff
@@ -80,15 +69,20 @@ const PatientNoteCard = ({
       return;
     }
 
-    updateNote(payload);
+    const { res } = await request(routes.updatePatientNote, {
+      pathParams: { patientId, noteId: note.id },
+      body: payload,
+    });
+    if (res?.status === 200) {
+      Success({ msg: "Note updated successfully" });
+      setIsEditing(false);
+      setReload(true);
+    }
```

Comment:
why were these changes even made? not sure why these are needed to solve these issue.

Reply:
I think it came out from develop branch when it is not up to date